lib/web_ui/lib/src/engine/semantics/text_field.dart

```diff
@@ -33,14 +33,14 @@ class SemanticsTextEditingStrategy extends DefaultTextEditingStrategy {
   /// This method must be called prior to accessing [instance].
   static SemanticsTextEditingStrategy ensureInitialized(
       HybridTextEditing owner) {
-    if (_instance != null && instance.owner == owner) {
-      return instance;
+    if (_instance != null && instance?.owner == owner) {
+      return instance!;
     }
     return _instance = SemanticsTextEditingStrategy(owner);
   }
 
   /// The [SemanticsTextEditingStrategy] singleton.
-  static SemanticsTextEditingStrategy get instance => _instance!;
+  static SemanticsTextEditingStrategy? get instance => _instance!;
```

Comment:
I would suggest keeping this non-nullable (i.e. remove the `?`).

And callsites should be updated to use the private nullable `_instance`:
```dart
SemanticsTextEditingStrategy._instance?.activate(this);
SemanticsTextEditingStrategy._instance?.deactivate(this);
```